docs/develop/develop-overview.md

```diff
@@ -68,13 +54,15 @@ The Office JavaScript API library can be accessed via the Office.js content deli
 > [!NOTE]
 > To use preview APIs, reference the preview version of the Office JavaScript library on the CDN: https://appsforoffice.microsoft.com/lib/beta/hosted/office.js.
 
+For more information about accessing the Office JavaScript library, including how to get IntelliSense if you're using TypeScript, see [Referencing the JavaScript API for Office library from its content delivery network (CDN)](referencing-the-javascript-api-for-office-library-from-its-cdn.md).
```

Comment:
I think developers can use type definitions to get Intellisense with JavaScript as well: https://docs.microsoft.com/en-us/visualstudio/ide/javascript-intellisense?view=vs-2019